src/test/scala/com/cognite/sdk/scala/v1/EdgesTest.scala

```diff
@@ -305,8 +305,8 @@ class EdgesTest extends CommonDataModelTestHelper with RetryWhile with BeforeAnd
         )
         .unsafeRunSync()
 
-      exception.message.contains("invalid input") shouldBe true
-      exception.message.contains("abc") shouldBe true
+      exception.message should contain ("invalid input")
```

Comment:
Slightly better if `EdgesTest` and `NodesTest` would have `with OptionValues` so we can use `.value` instead of `contain`, gives better error messages from ScalaTest. See https://www.scalatest.org/user_guide/using_OptionValues

Reply:
I think you mean for the case when `exception.message` is an optional field? Here we just check if a string contains a string or not 😅